docs/NooBaaNonContainerized/NooBaaCLI.md

```diff
@@ -59,14 +59,20 @@ The `account add` command is used to create a new account with customizable opti
 
 #### Usage
 ```sh
-noobaa-cli account add --name <account_name> --uid <uid> --gid <gid> [--user]
+noobaa-cli account add --name <root_account_name> --uid <uid> --gid <gid> [--user]
 [--new_buckets_path][--access_key][--secret_key][--fs_backend]
 [--allow_bucket_creation][--force_md5_etag][--anonymous][--from_file][--iam_operate_on_root_account]
 ```
 #### Flags -
 - `name` (Required)
     - Type: String
-    - Description: Specifies the name of the new account.
+    - Description: Specifies the name of the root account.
+    If creating a iam account, specify the name of the new iam account with --iam_name param.
+
+- `iam_name`
```

Comment:
the new flag should be added as optional with brackets, for example: `[--iam_name]`